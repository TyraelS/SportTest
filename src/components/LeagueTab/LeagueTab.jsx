import React from 'react';
import PropTypes from 'prop-types';

import LeagueTabWrapper from './LeagueTabWrapper.style';

const LeagueTab = ({ name, country }) => {
  return (
    <LeagueTabWrapper>
      <div>{name}</div>
      <div>{country}</div>
    </LeagueTabWrapper>
  );
};

LeagueTab.displayName = 'LeagueTab';
LeagueTab.propTypes = {
  name: PropTypes.string.isRequired,
  country: PropTypes.string.isRequired
};

export default LeagueTab;
