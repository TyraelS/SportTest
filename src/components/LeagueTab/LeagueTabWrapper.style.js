import styled from 'styled-components';

const LeagueTabWrapper = styled.div`
  display: flex;
  justify-content: space-between;
  align-items: center;
  > div {
    padding: 10px;
  }
`;

export default LeagueTabWrapper;
