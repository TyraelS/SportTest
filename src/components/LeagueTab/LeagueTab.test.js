import React from 'react';
import { shallow } from 'enzyme';
import LeagueTab from './LeagueTab';

describe('Given the LeagueTab component with provided props', () => {
  describe('when the component is rendered', () => {
    const component = shallow(<LeagueTab name='Test name' country='GBR' />);

    it('should match the snapshot', () => {
      expect(component).toMatchSnapshot();
    });
  });
});
