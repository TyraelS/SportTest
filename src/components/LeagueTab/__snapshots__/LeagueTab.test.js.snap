// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the LeagueTab component with provided props when the component is rendered should match the snapshot 1`] = `
<ForwardRef(styled.div)>
  <div>
    Test name
  </div>
  <div>
    GBR
  </div>
</ForwardRef(styled.div)>
`;
